lib/modules/apostrophe-areas/public/js/editor.js

```diff
@@ -63,12 +63,11 @@ apos.define('apostrophe-areas-editor', {
     };
 
     self.registerClickHandlers = function() {
-
       self.link('apos-add-item', self.startAutosavingHandler(self.addItem));
       self.link('apos-edit-item', self.startAutosavingHandler(self.editItem));
       self.link('apos-move-item', self.startAutosavingHandler(self.moveItem));
       self.link('apos-trash-item', self.startAutosavingHandler(self.trashItem));
-
+      self.$el.find('[data-apos-tooltip]').hover(apos.ui.createTooltip, apos.ui.removeTooltip);
```

Comment:
Since this isn't using delegation, it will not work for tooltips added to the page after initial page load, as is common when working with nested widgets.